target/linux/ath79/dts/qca9558_compex_wpj558-16m.dts

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "qca955x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "compex,wpj558-16m", "qca,qca9558";
+	model = "Compex WPJ558 (16MB flash)";
+
+	aliases {
+		label-mac-device = &eth0;
+		led-boot = &led_sig4;
+		led-failsafe = &led_sig4;
+		led-running = &led_sig4;
+		led-upgrade = &led_sig4;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		sig1 {
+			label = "red:sig1";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		sig2 {
+			label = "yellow:sig2";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		sig3 {
+			label = "green:sig3";
+			gpios = <&gpio 22 GPIO_ACTIVE_LOW>;
+		};
+
+		led_sig4: sig4 {
+			label = "green:sig4";
+			gpios = <&gpio 23 GPIO_ACTIVE_LOW>;
+		};
```

Comment:
Maybe call it status then like other boards... If the other LEDs have no other use, it could make sense to use the green LEDs as port indicators, the red one for upgrade and failsafe and the yellow one as running and boot or something...

Reply:
If they are originally signal strength indicators, using sig1 to sig4 is essentially fine. One should name them after their original purpose, and not rename just because we hijack them as system leds. So, based on that the current scheme is basically fine.

One optional improvement could be to include the strength into the name as some other device do, e.g. use green:signal-low, green:signal-mediumlow, green:signal-mediumhigh, green:signal-high or just green:low, green:mediumlow etc. But whether that's really better depends on certains aspects, and taste.

Reply:
Addendum: Since this actually an ar71xx device, I'd keep the sigX names in this case, as they are consistent with the old names and there is no strong reason to change.

Reply:
Ok il will finally keep the sigX name and replace led_sig4 by status

Reply:
No, please also keep led_sig4. This should be named after the led actually used, so from reading the lines in aliases node one can immediately understand which led is used.

Reply:
Ok thank you for the explanation